server/jetstream_jwt_test.go

```diff
@@ -1040,15 +1041,15 @@ func TestJetStreamDeletedAccountDoesNotLeakSubscriptions(t *testing.T) {
 	conf := createConfFile(t, []byte(fmt.Sprintf(`
 		listen: 127.0.0.1:-1
 		operator: %s
-		jetstream: {max_mem_store: 10Mb, max_file_store: 10Mb}
+		jetstream: {max_mem_store: 10Mb, max_file_store: 10Mb, store_dir: %v}
```

Comment:
whenever we do not set the `store_dir` in the tests, this seems to impact the startup of the tests for some reason, fixing it for this test here

Reply:
hmm wonder why?